### PR TITLE
NXBT-1046: Add the nightly profile

### PR DESCRIPTION
**Pull request by yjulienne-nuxeo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -713,6 +725,61 @@
...
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+      <repositories>
+        <repository>
+          <id>nightly-staging</id>
+          <url>http://mavenin.nuxeo.com/nexus/content/repositories/nightly-staging</url>
```
Comment (jcarsique) [inline pom.xml]: not blocker (fixed by Maven and Apache) but a preferred URL is
https://mavenin.nuxeo.com/nexus/content/repositories/nightly-staging/
with HTTPS and an ending slash

File: pom.xml [hunk 2]
```diff
@@ -713,6 +725,61 @@
...
+          </releases>
+          <snapshots>
+            <enabled>false</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+      <distributionManagement>
+        <repository>
+          <id>nightly-staging</id>
+          <url>http://mavenin.nuxeo.com/nexus/content/repositories/nightly-staging</url>
```
Comment (jcarsique) [inline pom.xml]: Same as above: HTTPS, ending slash

**Review by jcarsique — APPROVED** [review]
:+1:

**Review by kevinleturc — COMMENTED** [review]
👍

**Comment by atchertchian** [comment]
is this still valid?

**Pull request merged**